docs/pages/Theming/CustomTheme.md

```diff
@@ -34,23 +34,29 @@ components: {
 },
 ```
 
-For example, if you want to change the settings on the [Button component](https://github.com/microsoft/fluentui-react-native/blob/master/packages/components/Button/src/Button.tsx), you could do something like the following:
+For example, if you want to change the settings on the [Button component](https://github.com/microsoft/fluentui-react-native/blob/master/packages/components/Button/src/Button.settings.ts), you could do something like the following:
 
 ```ts
 components: {
   Button: {
-    backgroundColor: 'primaryButtonBackground', // Different semantic color
-    color: 'neutralBackground1', // Alias token
-    hovered: {
-      backgroundColor: theme.host.colors['AppShade10'], // Get brand color from host
-      color: theme.host.colors['Gray96'], // Get gray from host
-      borderColor: theme.host.palette.Bkg, // Get entry from palette from host
+    tokens: {
+      backgroundColor: 'primaryButtonBackground', // Different semantic color
+      color: 'neutralBackground1', // Alias token
     },
+    _overrides:{
+      tokens:{
+        hovered: {
+          backgroundColor: theme.host.colors['AppShade10'], // Get brand color from host
+          color: theme.host.colors['Gray96'], // Get gray from host
+          borderColor: theme.host.palette.Bkg, // Get entry from palette from host
+        },
+      },
+    }
```

Comment:
I kinda liked the new framework's styling structure better than the old for documentation 😅 but this is probably more useful since the old button is what is (presumably) used in devmain more.

Reply:
An aside, a simpler .settings structure for themability I think was one of the goals of the new framework.

Reply:
Yeah, I agree that the newer frameworks are better, but since the goal is to be friendly to users and the experimental button isn't exported right now, this seemed like a better example to use for the reason you mentioned.